### ISSUE TITLE
On device white screen

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#208]
Building the app with from the `52n-v2-upgrade` branch with:

```
ionic cordova build android
```
works after adding a `google-services.json` to the base and updating `com.google.gms:google-services` in `/platforms/android/build.gradle` to:

```
classpath 'com.google.gms:google-services:4.2.0' // google-services dependency from cordova-plugin-firebase
```

On device however, this build installs fine, but you only get a white screen after launching.

This is not the case when building the `V2` branch and installing on device.

**Comment by opeeters** [comment]
Building with:

```
ionic cordova build android  --prod --release
```

Give the following errors:

```
> ng run app:ionic-cordova-build:production --platform=android



ERROR in src/app/v2/components/user-notification-settings/user-notification-settings.component.html:6:28 - error TS2554: Expected 0 arguments, but got 1.

6       (ionFocus)="hasFocus($event)" (ionBlur)="blurFocus($event)"></ion-toggle>
                             ~~~~~~

  src/app/v2/components/user-notification-settings/user-notification-settings.component.ts:22:18
    22     templateUrl: './user-notification-settings.component.html',
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component UserNotificationSettingsComponent.
src/app/v2/components/user-notification-settings/user-notification-settings.component.html:6:58 - error TS2554: Expected 0 arguments, but got 1.

6       (ionFocus)="hasFocus($event)" (ionBlur)="blurFocus($event)"></ion-toggle>
                                                           ~~~~~~

  src/app/v2/components/user-notification-settings/user-notification-settings.component.ts:22:18
    22     templateUrl: './user-notification-settings.component.html',
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component UserNotificationSettingsComponent.
src/app/v2/screens/main-screen/main-screen.component.html:20:62 - error TS2445: Property 'detailData' is protected and only accessible within class 'MainScreenComponent' and its subclasses.

20         <app-information-item *ngFor="let detailDataPoint of detailData" [detailedDataPoint]="detailDataPoint"
                                                                ~~~~~~~~~~

  src/app/v2/screens/main-screen/main-screen.component.ts:16:18
    16     templateUrl: './main-screen.component.html',
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component MainScreenComponent.
src/app/v2/screens/main-screen/main-screen.component.html:55:41 - error TS2445: Property 'userSettingsService' is protected and only accessible within class 'MainScreenComponent' and its subclasses.

55       <map-component [currentLocation]="userSettingsService.selectedUserLocation" [phenomenon]="detailPoint.substance.phenomenon"></map-component>
                                           ~~~~~~~~~~~~~~~~~~~

  src/app/v2/screens/main-screen/main-screen.component.ts:16:18
    16     templateUrl: './main-screen.component.html',
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component MainScreenComponent.

[ERROR] An error occurred while running subprocess ng.

        ng run app:ionic-cordova-build:production --platform=android exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

**Comment by janschulte** [comment]
I fixed these build errors with 3f7cf3ead044fcbc3d6a2c73812c3d0f50828ec6.

After that I successfully checked the build out of a new cloned repo on my device. I also need the small adjustment in `/platforms/android/build.gradle`

**Comment by opeeters** [comment]
Seems to be OK now
